python/dolfinx/fem/forms.py

```diff
@@ -190,7 +190,7 @@ def form(
     dtype: npt.DTypeLike = default_scalar_type,
     form_compiler_options: typing.Optional[dict] = None,
     jit_options: typing.Optional[dict] = None,
-    entity_maps: dict[Mesh, np.typing.NDArray[np.int32]] = {},
+    entity_maps: dict[Mesh, npt.NDArray[np.int32]] = {},
```

Comment:
Note to self, we should revise this, as it is not safe to have default dictionaries.